server.js

```diff
@@ -1,15 +1,37 @@
-const express = require('express')
-const { indexHandler, sayHelloHandler } = require('./routes/handlers')
-
+const express = require("express");
+const pino = require("pino");
+const { indexHandler, sayHelloHandler } = require("./routes/handlers");
 
 const app = express();
 const port = 3000;
 
+const logger = pino({
+  transport: {
+    target: "pino-pretty",
+    options: {
+      translateTime: "SYS:dd-mm-yyyy HH:MM:ss", // formate the date and hour
```

Comment:
Always write in ISO8601 format. **DO NOT** write dd-mm-yyyy or mm-dd-yyyy as it is **NOT** sortable!

Reply:
To simplify the issue, use 24hrs format with UTC timezone unless there is a need to use localised timezone.